app/src/features/pricing/components/PremiumFeature/index.tsx

```diff
@@ -53,21 +55,17 @@ export const PremiumFeature: React.FC<PremiumFeatureProps> = ({
   const isUpgradePopoverEnabled = useFeatureIsOn("show_upgrade_popovers");
   const isIncentivizationEnabled = useIsIncentivizationEnabled();
 
+  const theme = useTheme();
+
   const isExceedingLimits = useMemo(
     () => features.some((feat) => !(getFeatureLimitValue(feat) && !checkIfFeatureLimitReached(feat, "reached"))),
     [features, getFeatureLimitValue, checkIfFeatureLimitReached]
   );
   const isBreachingLimit = features.some((feat) => checkIfFeatureLimitReached(feat, "reached"));
 
-  const hasCrossedDeadline = useMemo(() => new Date() > new Date("2023-11-30"), []);
-
   const handlePopoverSecondaryAction = useCallback(() => {
-    onUpgradeForFreeClickCallback();
-
-    if (!hasCrossedDeadline) {
-      trackUpgradeOptionClicked(SOURCE.USE_FOR_FREE_NOW);
-      onContinue();
-    } else {
+    if (isIncentivizationEnabled) {
```

Comment:
This flag also has a check of `isNewUser` (which I think is only for users after `2024-06-20`: [Ref](https://github.com/requestly/requestly/blob/master/app/src/features/incentivization/hooks/useIsIncentivizationEnabled.ts))

could not confirm from the internal ticket if this is intended so just pointing it out